wg-notebooks/charter.md

```diff
@@ -0,0 +1,39 @@
+# WG Notebooks Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Notebooks is responsible for the user experience around Notebooks and other IDEs and their integrations with Kubeflow.
+The WG covers launching and managing different types of Notebooks on Kubeflow as well as workflows that Notebook extensions enable.
```

Comment:
This sounds in line with the WG name, it is just the "other IDEs" that trigger my confusion

Reply:
Perhaps 'web IDE's'

Reply:
will rename to `other web IDEs` to make a little bit more clear